shaders/src/light_punctual.fs

```diff
@@ -128,7 +128,7 @@ Light getLight(const uint index) {
     // and populate the Light structure
     Light light;
     light.colorIntensity.rgb = colorIntensity.rgb;
-    light.colorIntensity.w = computePreExposedIntensity(colorIntensity.w, frameUniforms.exposure);
+    light.colorIntensity.w = computePreExposedIntensity(colorIntensity.w, getExposure());
```

Comment:
Sooo… here it would make more sense to keep accessing the uniform since this particular function will make the value highp again to do a highp multiplication.